pkg/controllers/disruption/controller.go

```diff
@@ -107,7 +115,8 @@ func (c *Controller) Reconcile(ctx context.Context) (reconcile.Result, error) {
 	c.recordRun("disruption-loop")
 
 	// Log if there are any budgets that are misconfigured that weren't caught by validation.
-	c.logInvalidBudgets(ctx)
+	// Only validate the first reason, since CEL validation will catch invalid disruption reasons
+	c.logInvalidBudgets(ctx, c.methods[0].Reason())
```

Comment:
This feels kind of hacky, can we catch this at the NodePool validation layer? I'm find with taking this iteratively, but I don't think that we should pass in a reason in the function signature

Reply:
Yeah, we can do that. I'll drop it in the function signature and relay on the values in the controller